tor_ocr/main.py

```diff
@@ -165,6 +169,7 @@ def decode_image_from_url(url, overlay=False, api_key=__OCR_API_KEY__):
     return result.json()
 
 
+# noinspection PyShadowingNames
```

Comment:
Rather than disabling the rule, shouldn't we fix it so the rule isn't triggered?

Reply:
Gonna have to agree here, I try to avoid shadowing names.

Reply:
It's complaining about config, which is broken anyway. There is nothing to shadow because it's a global object.